app/styles/main.scss

```diff
@@ -955,7 +958,7 @@ ul.main-nav li ul li a{
 .nav .open > a:focus{
   background: $theme-blue;
   color:#ffffff;
-  border:1px solid $theme-blue;
+  border:1px solid transparent;
```

Comment:
Hey @esoliani the border is transparent because without that the grey tab had a blue border on focus. On hover, focus, active the drop down is blue so that it's consistent with the other active tabs but let me know if you and @ddimunzio  don't like that.